mongodb/README.md

```diff
@@ -0,0 +1,249 @@
+```json
+{
+  "publication": "Romeo and Juliet",
+  "text": "What\'s in a name? That which we call a rose by any other word would smell as sweet..."
+}
+```
+
+When deploying the `StatefulSet`, the Operator creates a MongoDB Replica Set, where one `Pod` holds the primary data set and the other ones hold replicas. Let's first verify this by running the following command:
+
+```
+kubectl exec -it mongodb-0 -- mongosh --username admin --password mongo --quiet --eval 'printjson(rs.status())'
+```
+
+Identify the primary MongoDB node. You should see a JSON Key/Value pair like below:
+```
+stateStr: 'PRIMARY'
+```
+
+The following MongoDB commands must be run from the primary node. Replace the `Pod` ordinal number accordingly so you can create the database and populate the collection.
+
+```
+kubectl exec -it mongodb-0 -- mongosh admin < ./mongo.sh
+```
+
+Result:
+```
+...
+mongodb [direct: primary] admin> { ok: 1 }
+mongodb [direct: primary] admin> switched to db shakespeare
+mongodb [direct: primary] shakespeare> { ok: 1 }
+mongodb [direct: primary] shakespeare> {
+  acknowledged: true,
+  insertedId: ObjectId("6209a09540c27d9ba2907b76")
+}
+```
+
+Let's query the database to check that the document has been correctly created.
+```
+kubectl exec -it mongodb-0 -- mongosh admin < ./checkmongo.sh
+```
+
+Result:
+
+```
+...
+mongodb [direct: primary] admin> { ok: 1 }
+mongodb [direct: primary] admin> switched to db shakespeare
+mongodb [direct: primary] shakespeare> [
+  {
+    _id: ObjectId("6209a09540c27d9ba2907b76"),
+    publication: 'Romeo and Juliet',
+    text: "What's in a name? That which we call a rose by any other word would smell as sweet..."
+  }
+]
+```
+
+All good! You have deployed a new database that contains a document. Now, Let's verify that the database is encrypted!
+
+## Verify Volume Encryption
+First, let's find the Ondat volumes associated with the MongoDB `Pods` by executing the following commands:
+```
+# Be sure to change STORAGEOS_USERNAME and STORAGEOS_PASSWORD to match your configuration
+kubectl -n storageos create -f-<<END
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: storageos-cli
+  namespace: storageos
+  labels:
+    app: storageos
+    run: cli
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: storageos-cli
+      run: cli
+  template:
+    metadata:
+      labels:
+        app: storageos-cli
+        run: cli
+    spec:
+      containers:
+      - command:
+        - /bin/sh
+        - -c
+        - "while true; do sleep 3600; done"
+        env:
+        - name: STORAGEOS_ENDPOINTS
+          value: http://storageos:5705
+        - name: STORAGEOS_USERNAME
+          value: storageos
+        - name: STORAGEOS_PASSWORD
+          value: storageos
+        image: storageos/cli:v2.5.0
+        name: cli
+END
+POD=$(kubectl -n storageos get pod -ocustom-columns=_:.metadata.name --no-headers -lapp=storageos-cli)
+ kubectl port-forward svc/storageos -n storageos 5705 &
```

Comment:
No need to port-forward if we use the CLI, this bit can be snipped out!

Reply:
The CLI install is done by the kubectl commands immediately above this line, with `$POD` being set to the name of the container! Accepting both of the changes will result in a fully usable CLI install and usage for the command.

Reply:
Oh I see, you added the deployment config. I'm not a big fan of additional manual action if you don't need to. `kubectl port-forward` is good enough so I don't see why I'd add a new deployment for the CLI.

Reply:
Sure, in that case is it worth describing the install process for the storageos CLI locally? Port-forwarding won't install the CLI, after all.

Reply:
I can make a note pointing to the docs.